.github/ISSUE_TEMPLATE/proposal.md

```diff
@@ -14,6 +14,9 @@ Impact: Describe the customer impact of the problem. Who will this help?  How wi
 Scope: How much effort will this take? ok to provide a range of options if or "not yet determined" for initial proposals.  Feel free to include proposed tasks below or link a Google doc 
 
 TO DO
-- [ ] SIG Representative
-- [ ] Project leader(s)
-- [ ] TBD
+- [ ] Security TAG Leadership Representative:
+- [ ] Project leader(s):
+- [ ] Project Members:
+- [ ] _Fill in addition TODO items here so the project team and community can see progress!_
+- [ ] Project Schedule
```

Comment:
```suggestion
- [ ] Scope 
- [ ] Deliverable(s)
- [ ] Project Schedule
- [ ] Slack Channel (as needed)
- [ ] Meeting Time & Day:
- [ ] Meeting Notes (link)
- [ ] Meeting Details (zoom or hangouts link)
```

Reply:
More suggestions for the TODO.  This is the list we have for the controls catalog and is generally information useful to have on an issue once scope is defined.  It should also help potential project leads understand what goes into coordination for some efforts